lib/project.js

```diff
@@ -3,6 +3,9 @@ var git = require('gift');
 var normalizePackageData = require('normalize-package-data');
 var path = require('path');
 
+var BLIP_SCRIPT = 'node scripts/sl-blip.js';
```

Comment:
Oh, nevermind.. you are suggesting not using a folder at all. Hm.. sure, I could live with that.